pallets/fragments/src/weights.rs

```diff
@@ -45,86 +42,58 @@
-	// Storage: TransactionStorage StoragePeriod (r:1 w:0)
-	// Storage: TransactionStorage ChunkCount (r:1 w:0)
-	// Storage: System ParentHash (r:1 w:0)
-	// Storage: TransactionStorage Transactions (r:1 w:0)
-	fn check_proof_max() -> Weight {
-		(182_886_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(5 as Weight))
-			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	// Storage: Fragments Fragments (r:1 w:1)
+	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
+	// Storage: Fragments UnverifiedFragments (r:1 w:1)
+	fn upload() -> Weight {
+		(31_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(3 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
-	// Storage: TransactionStorage MaxTransactionSize (r:1 w:0)
-	// Storage: TransactionStorage ByteFee (r:1 w:0)
-	// Storage: TransactionStorage EntryFee (r:1 w:0)
-	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
-	// Storage: TransactionStorage BlockTransactions (r:1 w:1)
-	// Storage: TransactionStorage MaxBlockTransactions (r:1 w:0)
-	fn store(l: u32, ) -> Weight {
-		(0 as Weight)
-			// Standard Error: 0
-			.saturating_add((8_000 as Weight).saturating_mul(l as Weight))
-			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
+	// Storage: Fragments FragmentValidators (r:1 w:1)
+	fn add_validator() -> Weight {
+		(6_000_000 as Weight)
+			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
-	// Storage: TransactionStorage Transactions (r:1 w:0)
-	// Storage: TransactionStorage ByteFee (r:1 w:0)
-	// Storage: TransactionStorage EntryFee (r:1 w:0)
-	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
-	// Storage: TransactionStorage BlockTransactions (r:1 w:1)
-	// Storage: TransactionStorage MaxBlockTransactions (r:1 w:0)
-	fn renew() -> Weight {
-		(67_532_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
+	// Storage: Fragments FragmentValidators (r:1 w:1)
+	fn remove_validator() -> Weight {
+		(7_000_000 as Weight)
+			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
-	// Storage: TransactionStorage ProofChecked (r:1 w:1)
-	// Storage: TransactionStorage StoragePeriod (r:1 w:0)
-	// Storage: TransactionStorage ChunkCount (r:1 w:0)
-	// Storage: System ParentHash (r:1 w:0)
-	// Storage: TransactionStorage Transactions (r:1 w:0)
-	fn check_proof_max() -> Weight {
-		(182_886_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+	// Storage: Fragments Fragments (r:1 w:1)
+	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
+	// Storage: Fragments UnverifiedFragments (r:1 w:1)
+	fn upload() -> Weight {
```

Comment:
upload and also update need to be like
https://github.com/fragcolor-xyz/clamor/blob/e71d302957f3692dc38f27fe56932b35bb65760a/pallets/fragments/src/weights.rs#L64
Where you can pass the number of bytes and take that into account please.